source/_integrations/thread.markdown

```diff
@@ -14,30 +14,144 @@ ha_platforms:
+
+1. If you see the Matter logo on your device, follow the procedure [adding a Matter device to Home Assistant](/integrations/matter/#adding-a-matter-device-to-home-assistant).
+
+   <img src="/images/integrations/thread/matter_onpackbadge_logo.png"  width="200">
+
+2. If you see the Apple HomeKit logo on your device, follow the procedure [adding a HomeKit device to Home Assistant](/integrations/homekit_controller/#adding-a-homekit-device-through-thread).
+
+   <img src="/images/integrations/thread/apple-works-with-homekit-logo.png"  width="200">
+
+## About Thread
+
+This section introduces the terms *Thread* and *border router* and lists border routers that are supported by Home Assistant.
+
+### A communication protocol for IoT devices
+
+Thread is a low-power mesh networking standard for IoT devices. The low-power aspect is important for battery-powered smart home devices. However, it's also low-bandwidth, making it ideal for applications that don't send a lot of data, like switches or motion sensors.
+
+Thread uses the same <abbr title="radio frequency">RF</abbr> technology as Zigbee (IEEE 802.15.4) but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow controlling devices: It is just a communication protocol. To control the Thread devices, a higher-level protocol is required: Matter or Apple HomeKit.
+
+Thread devices use the IPv6 standard to communicate both inside and outside the mesh network.
+
+### About Thread border routers
+
+The devices use Thread border routers to communicate outside the mesh with any IPv6-capable device. A Thread border router is connected to your network either via Wi-Fi or Ethernet and uses its <abbr title="radio frequency">RF</abbr> radio to communicate with the Thread mesh network. The <abbr title="Thread border router">TBR</abbr> routes packets between your local network and the Thread mesh. It does not look at the content of these packets, it just forwards them.
+
+Often, Thread border routing is only an auxiliary functionality of a smart home device. For example, the Nest Hub (2nd gen) is a smart display, a Google Home controller, and a Chromecast target, but also has a Thread border router included.
+
+Unlike other protocols, Thread can use multiple border routers in a single network. This increases wireless coverage and reduces the risk of a single point of failure. Ideal for home automation, with a potentially large number of devices spread over a large area.
+
+Home Assistant can only control OpenThread <abbr title="Thread border routers">TBRs</abbr>. OpenThread is an open source implementation of Thread, originally released by Google.
+
+### List of Thread border router devices
+
+Currently, the following <abbr title="Thread border router">TBR</abbr> devices are known to work well with Home Assistant:
+
+#### Nabu Casa
+
+- [Home Assistant Yellow](/yellow/) hub
+- [Home Assistant SkyConnect](/skyconnect/) Zigbee/Thread USB stick
```

Comment:
Hm, maybe we should add a note that this is Zigbee by default along with a link how-to make it Thread? :thinking: